src/style.css

```diff
@@ -1,3 +1,87 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(https://fonts.google.com/specimen/Roboto);
+}
+
+html{
+  font-family: Roboto, sans-serif;
```

Comment:
You have correctly added a fallback font `sans-serif` for the `Roboto` font. However, remember to add a more specific fallback font before the generic `sans-serif` in case `Roboto` doesn't load. This is a good practice to ensure your design remains as consistent as possible across different browsers and operating systems.